launch/prop_follower.launch

```diff
@@ -2,24 +2,26 @@
 <launch>
   <!-- Specify what functionality you are testing-->
     <arg name="object_detection" default="false"/> 
-    <arg name="lidar-distance-angle-measurement" default="false"/> <!-- setting to false produces arbitrary fake coordinates-->
+    <arg name="lidar-distance-angle-measurement" default="true"/> <!-- setting to false produces arbitrary fake coordinates-->
     <arg name="velocity_sender" default="true"/> <!-- setting to true uses velocity_sender, setting to false uses waypoint_sender -->
 
   <!--=======================================================================-->
 
-  
+  <!--Launch file for sim-->
+  <include file="$(find box_bot_sim_pkg)/launch/rect_bot_world.launch"/>
```

Comment:
I think this could be in a separate launch file. It's convenient to have to here for testing but it isn't directly related to the prop_follower package. 

An idea would be to create a testing launch file that could be used when testing in sim. This file would also have the same content as the prop_follower with additional params for gazebo.

Reply:
Change made